torch_frame/data/mapper.py

```diff
@@ -291,7 +291,15 @@ def forward(
             tokenized_list: TextTokenizationOutputs = self.text_tokenizer(
                 ser_list)
             for key in tokenized_list[0]:
-                xs = [[item[key]] for item in tokenized_list]
+                xs = []
+                for item in tokenized_list:
+                    if item[key].ndim == 1:
+                        xs.append([item[key]])
+                    elif item[key].ndim == 2:
+                        xs.append([row for row in item[key]])
+                    else:
+                        raise ValueError(f'{key} has `ndim` not '
+                                         f'equal to 1 or 2.')
```

Comment:
Do you mind explaining the logic here a little bit? Thanks~

Reply:
It depends on for example what tensor is returned by tokenizer. When we have multiple sentences, tokenizer will return a tensor with dim=2, each row corresponds to a row. Sometimes tokenizer might return 1 dim tensor when input (minibatch size is 1) only has one sentence.